test/HaRePluginSpec.hs

```diff
@@ -64,16 +67,11 @@ hareSpec = do
                                                   ,("name",ParamTextP "foolong")])
       r <- dispatchRequest req
       r `shouldBe`
-        Just (IdeResponseOk (jsWrite (RefactorResult [HieDiff
-                                                           (cwd </> "test/testdata/HaReRename.hs")
-                                                           (cwd </> "test/testdata/HaReRename.refactored.hs")
-                                                           ("4,5c4,5\n"++
-                                                            "< foo :: Int -> Int\n"++
-                                                            "< foo x = x + 3\n"++
-                                                            "---\n"++
-                                                            "> foolong :: Int -> Int\n"++
-                                                            "> foolong x = x + 3\n")
-                                                           ])))
+        Just (IdeResponseOk $
+              jsWrite
+              $ RefactorResult
+              [WorkspaceEdit
+              {_changes = Just (H.fromList [(filePathToUri $ cwd </> "test/testdata/HaReRename.hs",List [TextEdit {_range = Range {_start = Position {_line = 3, _character = 0}, _end = Position {_line = 4, _character = 13}}, _newText = "foolong :: Int -> Int\nfoolong x = x + 3"}])]), _documentChanges = Nothing}] )
```

Comment:
Please format this and the other test results to not only be on one line, and that it is reasonably clear what is going on, as was done with the original.